ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/loopback_interfaces.schema.yml

```diff
@@ -0,0 +1,78 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  loopback_interfaces:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          description: Loopback Interface name like "Loopback0"
+          required: true
+        description:
+          type: str
+        shutdown:
+          type: bool
+        vrf:
+          type: str
+          description: VRF name
+        ip_address:
+          type: str
+          description: IPv4_address/Mask
+        ip_address_secondaries:
+          type: list
+          items:
+            type: str
+            description: IPv4_address/Mask
+        ipv6_enable:
+          type: bool
+        ipv6_address:
+          type: str
+          description: IPv6_address/Mask
+        ip_proxy_arp:
+          type: bool
+        ospf_area:
+          type: str
+          convert_types:
+          - int
+        mpls:
+          type: dict
+          keys:
+            ip:
+              type: bool
+            ldp:
+              type: dict
+              keys:
+                interface:
+                  type: bool
+        eos_cli:
+          type: str
+          description: EOS CLI rendered directly on the loopback interface in the final EOS configuration
```

Comment:
I would put this key at the end of the options as this is a bit particular - @ClausHolbechArista thoughts?

Reply:
fine for me